rest/route/distro.go

```diff
@@ -742,182 +579,6 @@ func validateDistro(ctx context.Context, apiDistro *model.APIDistro, resourceID
 	return d, nil
 }
 
-///////////////////////////////////////////////////////////////////////
-//
-// PATCH /rest/v2/distros/{distro}/execute
-
-type distroExecuteHandler struct {
```

Comment:
The CLI and [REST client](https://github.com/evergreen-ci/evergreen/blob/8db3a5c7fe497090c13f0411d2881eda5cc372eb/rest/client/methods.go#L678-L704) logic to use this route also needs to be deleted.

Reply:
done